app/main.py

```diff
@@ -1,13 +1,22 @@
+format_error = {
+    "line_number": "line", "column_number": "column", "text": "message",
+    "code": "name", "flake8": "source"
+}
+
+
 def format_linter_error(error: dict) -> dict:
-    # write your code here
-    pass
+    return {value: (error[key] if key in error else key)
+            for (key, value) in format_error.items()} if error else {}
 
 
 def format_single_linter_file(file_path: str, errors: list) -> dict:
-    # write your code here
-    pass
+    return {"errors": [format_linter_error(error) for error in errors],
+            "path": file_path,
+            "status": "failed" if errors else "passed"
+            }
```

Comment:
In case there are arguments on the same line with an opening parenthesis, your closing parenthesis should be placed on the same line with arguments as well. Please, resolve this issue in all places